lib/WebAssemblyParser.js

```diff
@@ -4,7 +4,8 @@
 */
 "use strict";
 
-const Tools = require("webassembly-interpreter/lib/tools");
+const traverse = require("@webassemblyjs/ast").traverse;
+const decode = require("@webassemblyjs/wasm-parser").decode;
```

Comment:
I would prefer `const {traverse} = require("@webassemblyjs/ast")` but it would have been inconsistent

Reply:
? 🙃 Just use it and if you like update the other imports aswell :P

Reply:
That looks intentional, probably because of < Node 4?

Reply:
Yep sure, but `webpack >= v4.0.0` has `node >= v6.0.0` as minimum, so the code is simply just not 'upgraded' here yet :)